Code/Engine/Foundation/Threading/Implementation/TaskWorkerThread.cpp

```diff
@@ -14,7 +14,7 @@ static const char* GenerateThreadName(ezWorkerThreadType::Enum ThreadType, ezUIn
 }
 
 ezTaskWorkerThread::ezTaskWorkerThread(ezWorkerThreadType::Enum ThreadType, ezUInt32 uiThreadNumber)
-  : ezThread(GenerateThreadName(ThreadType, uiThreadNumber), 32 * 1024) /* 32 KB of stack size */
+  : ezThread(GenerateThreadName(ThreadType, uiThreadNumber), 128 * 1024) /* 128 KB of stack size */
```

Comment:
Maybe add a comment why at least 128 KB are needed, for future generations to not try to reduce it again.